common/src/main/java/com/viaversion/viaversion/protocols/protocol1_13to1_12_2/metadata/MetadataRewriter1_13To1_12_2.java

```diff
@@ -53,28 +59,13 @@ protected void registerRewrites() {
             }
         }));
 
-        filter().type(EntityTypes1_13.EntityType.ENDERMAN).index(12).handler((event, meta) -> {
-            // Remap held block to match new format for remapping to flat block
-            int stateId = meta.value();
-            int id = stateId & 4095;
-            int data = stateId >> 12 & 15;
-            meta.setValue((id << 4) | (data & 0xF));
-        });
-
         filter().type(EntityTypes1_13.EntityType.WOLF).index(17).handler((event, meta) -> {
             // Handle new colors
             meta.setValue(15 - (int) meta.getValue());
         });
 
         filter().type(EntityTypes1_13.EntityType.ZOMBIE).addIndex(15); // Shaking
 
-        filter().type(EntityTypes1_13.EntityType.MINECART_ABSTRACT).index(9).handler((event, meta) -> {
```

Comment:
Needs to be added back since it's not an block state type.

Reply:
Fixed